pom.xml

```diff
@@ -242,6 +243,41 @@
                 <version>${coveralls-maven-plugin.version}</version>
             </plugin>
         </plugins>
+        <pluginManagement>
+        	<plugins>
+        		<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+        		<plugin>
+        			<groupId>org.eclipse.m2e</groupId>
+        			<artifactId>lifecycle-mapping</artifactId>
+        			<version>1.0.0</version>
+        			<configuration>
+        				<lifecycleMappingMetadata>
+        					<pluginExecutions>
+        						<pluginExecution>
+        							<pluginExecutionFilter>
+        								<groupId>
+        									org.apache.maven.plugins
+        								</groupId>
+        								<artifactId>
+        									maven-checkstyle-plugin
+        								</artifactId>
+        								<versionRange>
+        									[2.17,)
+        								</versionRange>
+        								<goals>
+        									<goal>checkstyle</goal>
+        								</goals>
+        							</pluginExecutionFilter>
+        							<action>
+        								<ignore></ignore>
```

Comment:
Hi @pderakhshanfar, I have checked in the old pom.xml and you are right. Probably I edited it via eclipse IDE because I had compilation errors; I removed the `<pluginManagement>` tag (from 246 to 280 line) and the compilation works. I also update the pom.xml.